application/locations-data-manager/app.py

```diff
@@ -1,83 +1,52 @@
 from chalice import Chalice
+import l_service
 
-#   import json   //Packages to be imported once Chalice is deployed
-import boto3
-
-#   import uuid   //Packages to be imported once Chalice is deployed
-#   from boto3.dynamodb.conditions import Key //Packages to be imported once Chalice is deployed
 
 app = Chalice(app_name="locations-data-manager")
-dynamodb = boto3.resource("dynamodb")
 
 
-@app.route("/locations", methods=["GET"], cors=True)
+@app.route("/locations", methods=["GET"])
 def get_locations():
-    #   request = app.current_request.json_body  //Required to get request from the API Gateway once it's set up
-
-    print("Get locations")
-
-    locations_table = dynamodb.Table("locations")
-
-    response = locations_table.get_item(
-        Key={
-            "id": "001",
-        }
-    )
+    print("Get locations record...")
+    request = app.current_request.json_body()
 
-    print(response["Item"])
-
-    print("\n\n\n-------\n\n\n")
+    l_id = request["id"]
+    print("Get l_id record...".l_id)
```

Comment:
Remove print - replace with debug if needed

Reply:
Have we thought about logging levels yet and a way to configure them?